dev/docker-compose.platform.yml

```diff
@@ -118,7 +158,7 @@ services:
       - jaeger
 
   localstack:
+    #<<: *loki-logged-service
     image: localstack/localstack
     ports:
       - "4566:4566"
-      - "4510-4559:4510-4559"
```

Comment:
removed as these are not needed for local development and were just hogging the tilt UI space